doc/charm++/manual.rst

```diff
@@ -2555,173 +2557,196 @@ Examples are in ``examples/charm++/load_balancing`` and
-
--  **GreedyRefineLB**: Uses a greedy algorithm that assigns the heaviest
-   object to the least loaded processor when the benefit outweighs the
-   migration cost, otherwise leaves the object on its current processor.
-   It takes an optional command-line argument *+LBPercentMoves*,which
-   specifies the percentage of migrations that can be tolerated.
-
--  **TopoCentLB**: Extends the greedy algorithm to take processor
-   topology into account.
-
--  **RefineLB**: Moves objects away from the most overloaded processors
-   to reach average, limits the number of objects migrated.
-
--  **RefineSwapLB**: Moves objects away from the most overloaded
-   processors to reach average. In case it cannot migrate an object from
-   an overloaded processor to an underloaded processor, it swaps objects
-   to reduce the load on the overloaded processor. This strategy limits
-   the number of objects migrated.
-
--  **RefineTopoLB**: Same idea as in RefineLB, but takes processor
-   topology into account.
-
--  **BlockLB**: This strategy does a blocked distribution of objects to
-   processors.
-
--  **ComboCentLB**: A special load balancer that can be used to combine
-   any number of centralized load balancers mentioned above.
-
-Listed below are some of the communication-aware load balancers:
-
--  **MetisLB**: Uses `METIS <http://glaros.dtc.umn.edu/gkhome/metis/metis/overview>`__
-   to partition the object communication graph.
-
--  **ScotchLB**: Uses the `SCOTCH <http://www.labri.fr/perso/pelegrin/scotch/>`__
-   library for partitioning the object
-   communication graph, while also taking object load imbalance into
-   account.
-
--  **GreedyCommLB**: Extends the greedy algorithm to take the
-   communication graph into account.
-
--  **RefineCommLB**: Same idea as in RefineLB, but takes communication
-   into account.
+TreeLB and its pluggable strategies supersede the previous
+implementations of centralized and hierarchical load balancing. To use
+TreeLB, the user selects one of several trees. Each level corresponds
+to a different division of the overall execution (e.g. PE,
+process). Each level is configurable with a list of strategies,
+frequency, and other parameters. See :numref:`treeLb` below for more
+detail, along with configuration and execution instructions. The
+following strategies can be used with TreeLB (the old runtime
+selection syntax still works and is specified in parentheses):
+
+- **Greedy**: Uses a greedy algorithm that iterates over the objects
+  and assigns the heaviest remaining object to the least loaded
+  processor. (Old: ``+balancer GreedyLB``)
+
+- **GreedyRefine**: Uses a greedy algorithm that assigns the heaviest
+  remaining object to the least loaded processor when it is currently
+  assigned to a heavily loaded processor, otherwise leaves the object
+  on its current processor to limit migrations. It takes an optional
+  argument ``tolerance``, which specifies the tolerance it should allow
```

Comment:
Can you add a little more detail on this option? Is it like "+tolerance 1.1" to set it 10% higher?

Reply:
Done.